### PR TITLE
Update alert-grouping.md

### PR DESCRIPTION
**Pull request by gchairuangsang**
Updating more disallowed fields

## Purpose of this pull request

This pull request added more disallowed fields.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)
[SUMO-237777](https://sumologic.atlassian.net/browse/SUMO-237777)

### CODE REVIEW

File: docs/alerts/monitors/alert-grouping.md [hunk 1]
```diff
@@ -157,3 +157,5 @@ For example, let's say that you configured a monitor to generate one alert per c
 #### What fields are not allowed for Alert Grouping?
 
 Fields with very high cardinality such as `_blockid`, `_raw`, `_messagetime`, `_receipttime`, and `_messageid` are not allowed for Alert Grouping.
+
+Fields from aggregate operators such as `_count`, `_avg`, and `_sum` are not allowed.
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-grouping.md:161]: Hi @gchairuangsang , is it ok with you if we provide the link to those operator docs?

```suggestion
Fields from aggregate operators such as [`_count`](/docs/search/search-query-language/group-aggregate-operators/count-count-distinct-and-count-frequent/#count), [`_avg`](/docs/search/search-query-language/group-aggregate-operators/avg), and [`_sum`](/docs/search/search-query-language/group-aggregate-operators/sum) are not allowed.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
suggestion

File: docs/alerts/monitors/alert-grouping.md (hunk 1 above)
Comment (gchairuangsang) [reply docs/alerts/monitors/alert-grouping.md:161]: absolutely, thank you!

**Review by kimsauce — APPROVED** [review]

File: docs/alerts/monitors/alert-grouping.md (hunk 1 above)
Comment (kimsauce) [reply docs/alerts/monitors/alert-grouping.md]: Ok cool! I'll push this live now. Thanks for your edit!

**Pull request merged**